tests/integration/Makefile

```diff
@@ -0,0 +1,48 @@
+SPIN?=spin
+
+# We are testing the sources, not the installed application
+SDL2PROMELA=PYTHONPATH=../../ python3 -m sdl2promela.sdl2promela
```

Comment:
maybe consider venv or tox to test also the installation process?

Reply:
tox.ini needs to be placed where setup.py is, so outside of tests. The command line to run is a part of the tox.ini file, so I would have to either put the tests into tox.init (which is not nice), or, to keep the current structure, do a redirection from makefile to tox to makefile. I am not sure whether this is worth the hassle.

Reply:
https://jira.n7space.com/browse/TMC-76